File: content/polympiads/2.temps-avant-le-terminus/codes/solution.cpp
#include <vector>
#include <iostream>

using namespace std;

int main () {
    int nbArrets;
    cin >> nbArrets;
    vector<int> durees(nbArrets - 1);
    for (int i = 0; i < durees.size(); i ++) {
        cin >> durees[i];
    }

    int tempsAuTerminus = 0;
	for (int iTrajet = 0; iTrajet < nbArrets - 1; iTrajet ++) {
        tempsAuTerminus += durees[iTrajet];
    }
		
	cout << tempsAuTerminus << "\n";
}

File: content/polympiads/2.temps-avant-le-terminus/codes/solution.py
nb_arrets = int(input())
durees = list(map(int, input().split()))

# On calcule la somme des éléments dans le tableau durees, qui nous donne
# Le résultat attendu.
temps_au_terminus = 0
for i_arret in range(nb_arrets - 1):
    temps_au_terminus += durees[i_arret]

# On affiche la durée
print(temps_au_terminus)